core/src/avm2/globals/flash/text/static_text.rs

```diff
@@ -1,17 +1,18 @@
-use crate::avm2::{Activation, Error, Object, Value};
+use crate::avm2::error::argument_error;
+use crate::avm2::{Activation, ClassObject, Error, Object, Value};
 use crate::avm2_stub_getter;
 
-pub fn native_instance_init<'gc>(
+pub fn static_text_allocator<'gc>(
```

Comment:
Won't a class extending `StaticText` hit this error when it tries to construct? IIRC that was the point of having a native instance initializer.

Reply:
`StaticText` is final, so that can't happen.